modules/hydra-master-main.nix

```diff
@@ -72,6 +72,7 @@ let
       { jobset = "cardano-rt-view"; }
       { jobset = "cardano-shell"; }
       { jobset = "cardano-wallet"; }
```

Comment:
This line should be swapped with the one below it due to the regex matching.  See comments in this section and at the top of this block.